pulpcore/cli/common/openapi.py

```diff
@@ -157,6 +157,26 @@ def extract_params(
             )
         return result
 
+    def validate_body(self, schema: Any, body: Dict[str, Any], uploads: Dict[str, bytes]) -> None:
+        schema_ref = schema.get("$ref")
+        if schema_ref:
+            if not schema_ref.startswith("#/components/schemas/"):
+                raise OpenAPIError(_("Api spec is invalid."))
+            schema_name = schema_ref[21:]
```

Comment:
Magic numbers make my teeth hurt, but I know that sometimes they have to stay magic. Can we get at least a comment here describing where in the schema_ref we're getting "21"?